input.js

```diff
@@ -0,0 +1,105 @@
+var path = require('path');
+var util = require('util');
+var Promise = require('promise');
+var Command = require('./command');
+var config = require('./config.json');
+
+// Cache whether the orangutan binary has been pushed to the device during
+// current session
+var installed = false;
+
+/**
+ * API for interacting with a device via touch mechanisms
+ * @param {Device} device
+ * @constructor
+ */
+var Input = function(device) {
+  this.serial = device.serial;
+  this.inputEvent = device.config.inputEvent;
+  this.scriptTimeout = device.config.scriptTimeout || 250;
```

Comment:
Should be "scriptWriteTimeout" i.e.:

 this.scriptTimeout = device.config.scriptWriteTimeout || 250;

Unless you change the corresponding key in config.json.